app/helpers/date_helper.rb

```diff
@@ -0,0 +1,39 @@
+module DateHelper
+  def format_date_of_birth(date)
+    I18n.l(date_from_string_or_date(date), format: :date_of_birth)
+  end
+
+  def format_date_of_visit(date)
+    I18n.l(date_from_string_or_date(date), format: :date_of_visit)
+  end
+
+  def format_time_str(time)
```

Comment:
Seems as we have a method called `format_time_str_24 ` would it make sense to call this `format_time_str_12`? Current method name is a little on the ambiguous side.